src/components/ProfilePage.js

```diff
@@ -98,83 +100,263 @@ export default function ProfilePage() {
+                                <Carousel.Caption>
+                                <h1>Contact Me</h1>
+                                <p>{email}</p>
+                                </Carousel.Caption>
+                            </Carousel.Item>
+                        </Carousel>
+                    </Row>       
+                    <Row>
+                        <Col xs lg="2">
+                            <h1 className="text-info">Skills</h1>
+                            <div className="skills-scroll">
+                            {skills2.map((skill) => {
+                                return (
+                                    <ListGroup>
+                                        <ListGroup.Item>{skill}</ListGroup.Item>
+                                    </ListGroup>
+                                );
+
+                            })
+                            }  
+                            </div>
+                            <h1 className="text-info">Interests</h1>
+                            <div className="interest-scroll">
+                            {interests2.map((interest) => {
+                                return (
+                                    <ListGroup>
+                                        <ListGroup.Item>{interest}</ListGroup.Item>
+                                    </ListGroup>
+                                );
+
+                            })
+                            } 
+                            </div>                   
+                        </Col>
+                        <Col md="auto">
+                        <h1 className="text-info">Links</h1>
+                            {links.map((link) => {
+                                return (
+                                    <Card style={{ width: '18rem' }}>
+                                    <Card.Body>
+                                        <Card.Title>{link.type}</Card.Title>
+                                        <Card.Link className="mb-2 text-muted" href={link.link}>{link.link}</Card.Link>
+                                        
+                                    </Card.Body>
+                                    </Card>
+                                );
+                            })
+                            }
+                        </Col>
+                        <Col md="auto">
+                            <h1 className="text-info">Project Following</h1>
+                            <div className="project-following-scroll">
+                            {projectFollowings.map((projectFollowing) => {
+                                return (
+                                    <Card>
+                                    <Card.Body>
+                                        <Card.Title>{projectFollowing.projectName}</Card.Title>
+                                        <Card.Text>
+                                        The project id is: {projectFollowing.projectId}
```

Comment:
If you end up getting rid of it, get rid of it here, too. Make sure to keep changes consistent.